mailbox/api/src/test/java/org/apache/james/mailbox/events/ErrorHandlingContract.java

```diff
@@ -51,6 +51,11 @@ private ThrowingListener() {
             timeElapsed = new ArrayList<>();
         }
 
+        @Override
+        public boolean isUsed(Event event) {
+            return true;
```

Comment:
Why overriding the method if it does the exact same as the defined default one?

Reply:
* Having a clear/straightforward code
 * Reducing coupling
 * ...

Reply:
Yes, i think it's preferable to not depend on the default implementation.
In fact i would even prefer to remove the default implementation on the interface, i kept it only because the MailboxListener is used as a Functional interface.